src/backend/commands/createas.c

```diff
@@ -642,6 +570,86 @@ rewriteQueryForIMMV(Query *query, List *colNames)
 	return rewritten;
 }
 
+/*
+ * makeIvmAggColumn -- make aggregation column which is added by ivm
+ */
```

Comment:
It seems good, but, for making this clear, could you please add explanation for the arguments?

Reply:
Thank you for comment. I added explanation please check it.

Reply:
Thank you for your comments and an exmple. I will fix this information